packages/internals/src/migrateTypes.ts

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line @typescript-eslint/no-namespace
+export namespace MigrateTypes {
```

Comment:
Nit: technically, I think it's better to name like "SchemaEngineTypes", but not blocking this PR for sure

Reply:
I see where you're coming from, and in principle, I'd agree with you. However, considering we still have the `@prisma/migrate` package, wouldn't that be confusing to have conflicting naming conventions within the same codebase?